goalgorilla/open_social
# Issue #2896916 by rolki: Fix issue with #lazy_builder and ajax comments for nodes

## Problems
- The social_ajax_comments module doesn't work with #lazy_builder
- We display only the first 50 comments in topics and events

## Solution
Remove pager since we don't have a pager for comments and we display only 50 first items. After the fix, we will display all comments and it's not a problem because the list of comments will be rendered by #lazy_builder.

## Issue tracker
https://www.drupal.org/project/ajax_comments/issues/2896916#comment-13845503 

## How to test
- Enable BigPipe and social_ajax_comments modules
- Go to any topic or event and try to leave a comment
- The commenting will be failing because of a pager (see screenshot bellow) 

## Screenshots
![Снимок экрана 2020-10-02 в 15 53 17](https://user-images.githubusercontent.com/50984627/95210615-5b4b2c80-07f4-11eb-992d-1061464fbb65.png)
